src/pages/linux/install-and-configure-apache-server-in-linux/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Install and Configure Apache Server in Linux
+---
+
+<div class="para">Apache HTTP Server, colloquially called Apache, is free and open-source 
+cross-platform web server software.
+Apache is developed and maintained by an 
+open community of developers under the auspices of the Apache Software Foundation.</div>
+
+# Install and Configure Apache Server in Linux
+
+Step1: Install Apache Server<br>
+```#yum install httpd```
```

Comment:
A simpler we of writing this in md would be
```sh
`yum install httpd`
```

We have also found that indication a bash/shell command with a `#` or `$` can be confusing to newer developers. Please remove them